Sources/Core/API/Label.swift

```diff
@@ -15,173 +15,185 @@ import QuartzCore
+    actionManager.deactivate()
+    pluginManager.deactivate()
+}
 
-    /// Remove the label from its scene
-    public func remove() {
-        scene?.remove(self)
-        scene = nil
-        layer.removeFromSuperlayer()
-    }
+// MARK: - Public
 
-    // MARK: - Private
+/// Remove the label from its scene
+public func remove() {
+    scene?.remove(self)
+    scene = nil
+    layer.removeFromSuperlayer()
+}
 
-    private func positionDidChange(from oldValue: Point) {
-        guard position != oldValue else {
-            return
-        }
+// MARK: - Private
 
-        layer.position = position
-        scene?.labelRectDidChange(self)
+private func positionDidChange(from oldValue: Point) {
+    guard position != oldValue else {
+        return
     }
 
-    private func sizeDidChange(from oldValue: Size) {
-        guard size != oldValue else {
-            return
-        }
+    layer.position = position
+    scene?.labelRectDidChange(self)
+}
 
-        layer.bounds.size = size
-        scene?.labelRectDidChange(self)
+private func sizeDidChange(from oldValue: Size) {
+    guard size != oldValue else {
+        return
     }
 
-    private func textDidChange() {
-        layer.string = text
-        autoResize()
-    }
+    layer.bounds.size = size
+    scene?.labelRectDidChange(self)
+}
 
-    private func fontDidChange() {
-        layer.font = font
-        layer.fontSize = font.pointSize
-        autoResize()
-    }
+private func textDidChange() {
+    layer.string = text
+    autoResize()
+}
 
-    private func scaleDidChange(from oldValue: Metric) {
-        guard oldValue != scale else {
-            return
-        }
+private func fontDidChange() {
+    layer.font = font
+    layer.fontSize = font.pointSize
+    autoResize()
+}
 
-        layer.scale = scale
+private func rotationDidChange(from oldValue: Metric) {
+    guard rotation != oldValue else {
+        return
     }
 
-    private func autoResize() {
-        guard shouldAutoResize else {
-            return
-        }
+    layer.rotation = rotation
+
+    events.rotated.trigger()
+}
+
+private func scaleDidChange(from oldValue: Metric) {
+    guard oldValue != scale else {
+        return
+    }
 
-        let objCString = NSString(string: text)
+    layer.scale = scale
+}
 
-        let rect = objCString.boundingRect(with: .zero,
-                                           options: [.usesLineFragmentOrigin],
-                                           attributes: [.font: font],
-                                           context: nil)
-        size = rect.size
+private func autoResize() {
+    guard shouldAutoResize else {
+        return
     }
 
+    let objCString = NSString(string: text)
+
+    let rect = objCString.boundingRect(with: .zero,
+                                       options: [.usesLineFragmentOrigin],
+                                       attributes: [.font: font],
+                                       context: nil)
+    size = rect.size
+}
+
```

Comment:
Looks like everything from here to the class declaration lost an indentation level and should be shifted to the right by 4 spaces. Other than that the code looks great!